content/docs/reference/version-policy.md

```diff
@@ -0,0 +1,229 @@
+        <td>Experimental</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="https://github.com/kubeflow/mxnet-operator">MXNet operator</a>
+          (<a href="/docs/components/training/mxnet/">docs</a>)</td>
+        <td>Experimental</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="https://github.com/kubeflow/kubeflow/tree/master/components/jupyter-web-app">Notebook 
+        web app</a> (<a href="/docs/notebooks/">docs</a>)</td>
+        <td>Stable</td>
+        <td>1.0</td>
+      </tr>
+      <tr>
+        <td><a href="https://github.com/kubeflow/kubeflow/tree/master/components/notebook-controller">Notebook
+        controller</a> (<a href="/docs/notebooks/">docs</a>)</td>
+        <td>Stable</td>
+        <td>1.0</td>
+      </tr>
+      <tr>
+        <td><a href="https://github.com/kubeflow/pipelines">Pipelines</a>
+          (<a href="/docs/pipelines/pipelines-quickstart/">docs</a>)</td>
+        <td>Beta</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="https://github.com/kubeflow/kubeflow/tree/master/components/profile-controller">Profile 
+        Controller</a> (<a href="/docs/other-guides/multi-user-overview/">docs</a>)</td>
+        <td>Stable</td>
+        <td>1.0</td>
+      </tr>
+      <tr>
+        <td><a href="https://github.com/kubeflow/pytorch-operator">PyTorch operator</a> (<a href="/docs/components/training/pytorch/">docs</a>)</td>
+        <td>Stable</td>
+        <td>1.0</td>
+      </tr>
+      <tr>
+        <td>PyTorch serving 
+          (<a href="/docs/components/serving/pytorchserving/">docs</a>)</td>
+        <td>Experimental</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="https://github.com/kubeflow/batch-predict">TensorFlow 
+          batch prediction</a>
+          (<a href="/docs/components/serving/tfbatchpredict/">docs</a>)</td>
+        <td>Experimental</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>TensorFlow serving (<a href="/docs/components/serving/tfserving_new/">docs</a>)</td>
+        <td>Experimental</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="https://github.com/kubeflow/tf-operator">TFJob operator</a>
+          (<a href="/docs/components/training/tftraining/">docs</a>)</td>
+        <td>Stable</td>
+        <td>1.0</td>
+      </tr>
+      <tr>
+        <td><a href="https://github.com/kubeflow/xgboost-operator">XGBoost operator</a></td>
+        <td>Experimental</td>
+        <td></td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+<a id="sdk-matrix"></a>
+## Kubeflow SDKs and CLIs
+
+Alongside Kubeflow {{% kf-latest-version %}}, you may want to use 
+one of the following Kubeflow SDKs and command-line interfaces 
+(CLIs).
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>SDK / CLI</th>
+        <th>Status in Kubeflow {{% kf-latest-version %}}</th>
```

Comment:
Nit: I think kfctl is included with Kubeflow v1.0.0, but Fairing and the Pipelines SDK must be installed separately, right? Saying `Status in Kubeflow v1.0.0` seems a bit odd to me, then.

Maybe something like `Status at time of Kubeflow v1.0.0 release` would be more precise … a bit wordy though.

Reply:
Thanks! I've changed "in" to "with".